plugins/node/opentelemetry-instrumentation-connect/src/instrumentation.ts

```diff
@@ -180,4 +212,33 @@ export class ConnectInstrumentation extends InstrumentationBase<Server> {
       return original.apply(this, args as UseArgs2);
     };
   }
+
+  public _patchHandle(original: Server['handle']): Server['handle'] {
+    const instrumentation = this;
+    return function (
+      this: Server,
+      req: PatchedRequest,
+      res: ServerResponse,
+      out?: Function
+    ): ReturnType<Server['handle']> {
+      addNewStackLayer(req);
+
+      function completeStack() {
+        req[_LAYERS_STORE_PROPERTY].pop();
+      }
+      const done = out
+        ? instrumentation._patchOut(out as NextFunction, completeStack)
+        : undefined;
+
+      return original.apply(this, [req, res, done as Function]);
+    };
+  }
+
+  public _patchOut(out: NextFunction, completeStack: () => void): NextFunction {
+    return function nextFunction(this: NextFunction, err?: any): void {
```

Comment:
```suggestion
    return function nextFunction(this: NextFunction, ...args: any[]): void {
```
and below you could use `Reflect.apply(out, this, args);`
or even use `Reflect.apply(out, this, arguments);`